exercises/matrix/example.scala

```diff
@@ -1,5 +1,9 @@
 case class Matrix(matrix: String) {
   lazy val rows: Vector[Vector[Int]] = matrix.lines.map(_.split(' ').map(_.toInt).toVector).toVector
 
-  lazy val cols: Vector[Vector[Int]] = rows.transpose
+  lazy val columns: Vector[Vector[Int]] = rows.transpose
```

Comment:
I love Scala's transpose function :) F# sorely misses this.

Reply:
One of these days I need to teach myself F#. Hasn't made it to the top of my stack yet:)

Reply:
Let me know when you do :)